spec/features/mechanics/show_spec.rb

```diff
@@ -0,0 +1,21 @@
+require "rails_helper"
+
+RSpec.describe "Mechanic Show Page" do
+  before :each do
+    @amusement_park = AmusementPark.create!(name: "Six Flags", admission_cost: 75)
+    @ride1 = @amusement_park.rides.create!(name: "The Hurler", thrill_rating: 7, open: false)
+    @ride2 = @amusement_park.rides.create!(name: "The Boardwalk", thrill_rating: 5, open: true)
+    @mechanic = Mechanic.create!(name: "Kara Smith", years_experience: 11)
+    @mechanic.rides << @ride1
+    @mechanic.rides << @ride2
+  end
+
+  it "should show the mechanic, their years of experience and, all the rides they're working on" do
+    visit mechanic_path(@mechanic)
+
+    expect(page).to have_content(@mechanic.name)
+    expect(page).to have_content(@mechanic.years_experience)
```

Comment:
Remember that we want to try to be more explicit with where exactly this content should appear on the page. For example, the 7 that is tied to this mechanics years of experience could be appearing in the footer in the address info, but it's a 7 so this test would pass. To be more explicit we can use labels: expect(page).to have_content("Years of Experience: #{@mechanic_1.years_experience}") and within blocks.